pkg/osm/types_test.go

```diff
@@ -0,0 +1,107 @@
+package osm
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"sort"
+	"strings"
+	"testing"
+
+	tassert "github.com/stretchr/testify/assert"
+)
+
+func TestEnvoyConfigParser(t *testing.T) {
+	assert := tassert.New(t)
+	actual := getReleases()
+
+	assert.Equal(actual, []string{"v0.5", "v0.6", "v0.7", "v0.8", "v0.9"})
+
+	for _, release := range actual {
+		controllerVersion := ControllerVersion(release)
+		{
+			_, exists := SupportedIngress[controllerVersion]
+			assert.Truef(exists, "IngressVersion does not contain info on OSM release %s", release)
+		}
+
+		{
+			_, exists := SupportedTrafficTarget[controllerVersion]
+			assert.Truef(exists, "SupportedTrafficTarget does not contain info on OSM release %s", release)
+		}
+
+		{
+			_, exists := SupportedTrafficSplit[controllerVersion]
+			assert.Truef(exists, "SupportedTrafficSplit does not contain info on OSM release %s", release)
+		}
+
+		{
+			_, exists := SupportedHTTPRouteVersion[controllerVersion]
+			assert.Truef(exists, "SupportedHTTPRouteVersion does not contain info on OSM release %s", release)
+		}
```

Comment:
@draychev Can we merge https://github.com/openservicemesh/osm-health/pull/16, then add the check for supported annotations here, before merging this?

Reply:
Ok I'll add them to my PR #16 since this has been merged.